store_test.js

```diff
@@ -0,0 +1,28 @@
+const { emailField } = require("./pages/auth");
+
+Feature('store');
+
+xScenario('create account', ({ I, homePage, authPage, createAccountPage }) => {
+    I.openStore();
+    I.see('Women');
```

Comment:
remove this I.see, we don't need to verify page here.
or move it to home page object